src/Jellyfin.XmlTv/XmlTvReader.cs

```diff
@@ -14,7 +14,8 @@ namespace Jellyfin.XmlTv
     // Reads an XmlTv file
     public class XmlTvReader
     {
-        private const string DateWithOffsetRegex = @"^(?<dateDigits>[0-9]{4,14})(\s(?<dateOffset>[+-]*[0-9]{1,4}))?$";
+        private const string DateWithOffsetRegex = @"^(?<dateDigits>[0-9]{4,14})(\s(?<dateOffset>[+-]*[0-9:]{1,5}))?$";
```

Comment:
This doesn't seem ideal. `:::::` would be valid

Reply:
Your right, let me take a look

Reply:
OK, regex and test cases updated